include/seqan3/alphabet/quality/phred42.hpp

```diff
@@ -0,0 +1,313 @@
+
+    /*!\brief Return the letter's numeric phred code.
+     *
+     * \details
+     *
+     * Satisfies the seqan3::detail::quality_concept::to_phred() requirement via the seqan3::to_phred() wrapper.
+     *
+     * \par Complexity
+     *
+     * Constant.
+     *
+     * \par Exceptions
+     *
+     * Guaranteed not to throw.
+     */
+    constexpr phred_type to_phred() const noexcept
+    {
+        return _value;
+    }
+
+    /*!\brief Return the letter's rank in the quality alphabet.
+     *
+     * \details
+     *
+     * Satisfies the seqan3::semi_alphabet_concept::to_rank() requirement via the seqan3::to_rank() wrapper.
+     *
+     * \par Complexity
+     *
+     * Constant.
+     *
+     * \par Exceptions
+     *
+     * Guaranteed not to throw.
+     */
+    constexpr rank_type to_rank() const noexcept
+    {
+        return _value;
+    }
+    //!\}
+
+    /*!\name Write functions
+     * \{
+     */
+     /*!\brief Assign from a character.
+      *
+      * \details
+      *
+      * Satisfies the seqan3::alphabet_concept::assign_char() requirement via the seqan3::assign_char() wrapper.
+      *
+      * \par Complexity
+      *
+      * Constant.
+      */
+    constexpr phred42 & assign_char(char_type const c)
+    {
+        assert(c >= offset_char && c < offset_char + max_value_size);
+        _value = char_to_value[c];
+        return *this;
+    }
+
+    /*!\brief Assign from the numeric phred value.
+     *
+     * \details
+     *
+     * Satisfies the seqan3::quality_concept::assign_phred() requirement via the seqan3::assign_rank() wrapper.
+     *
+     * \par Complexity
+     *
+     * Constant.
+     */
+    constexpr phred42 & assign_phred(phred_type const p)
+    {
+        // p >= 0 is implicitly always true
+        assert(p < max_value_size);
+        // round to largest allowed value when `p` exceeds maximal rank value
+        _value =  (p < value_size) ? p : value_size - 1;
+        return *this;
+    }
+
+    /*!\brief Assign from a the numeric rank value.
+     *
+     * \details
+     *
+     * Satisfies the seqan3::semi_alphabet_concept::assign_rank() requirement via the seqan3::assign_rank() wrapper.
+     *
+     * \par Complexity
+     *
+     * Constant.
+     *
+     * \par Exceptions
+     *
+     * Guaranteed not to throw.
+     */
+    constexpr phred42 & assign_rank(phred_type const p)
+    {
+        return assign_phred(p);
+    }
+    //!\}
+
+    /*!\name Conversion operators
+     * \{
+     */
+    //!\brief Implicit conversion between dna* and rna* of the same size.
+    //!\tparam other_nucl_type The type to convert to; must satisfy seqan3::quality_concept and have the same \link value_size \endlink.
+    template <typename other_qual_type>
+    //!\cond
+        requires quality_concept<other_qual_type> && value_size == alphabet_size_v<other_qual_type>
+    //!\endcond
+    constexpr operator other_qual_type() const //noexcept
+    {
+        return other_qual_type{_value};
+    }
```

Comment:
I think this can be removed since there are no other quality alphabets of the same size (and the description is wrong).